skimage/restoration/tests/test_denoise.py

```diff
@@ -414,8 +427,12 @@ def test_wavelet_denoising_nd():
             noisy = img + sigma * rstate.randn(*(img.shape))
             noisy = np.clip(noisy, 0, 1)
 
-            # Verify that SNR is improved with internally estimated sigma
-            denoised = restoration.denoise_wavelet(noisy, method=method)
+            # I don't know why this logic for the warnings is correct
```

Comment:
:rofl: This comment should be here.

Reply:
No, really, this should be removed :slightly_smiling_face: . Or, at least, refactored.

Reply:
@grlee77 do you have an insight of this?

Reply:
@soupault I added a little more information that I could gain.